test-cases/scylla-operator/longevity-scylla-operator-3h-minikube.yaml

```diff
@@ -16,5 +16,5 @@ n_monitor_nodes: 1
 nemesis_class_name: 'KubernetesScyllaOperatorMonkey'
 nemesis_interval: 5
 
-user_prefix: 'k8s-longevity-10gb-3h-minikube'
+user_prefix: 'k8s-longevity-10gb-3h'
```

Comment:
And , long-10gb-3h-mkube

Reply:
Know it looks like `k8s-longevity-10gb-3h-jenkins-k8s-minikube-c9cdbe63`. Good enough. We include backend name to the prefix